sql/files/defaultdata/java_javac_detect/run

```diff
@@ -76,13 +63,8 @@ if [ -z "$MAINCLASS" ]; then
 
 	# Report the entry point, so it can be saved, e.g. for later replay:
 	echo "Info: detected entry_point: $MAINCLASS"
-else
-	cat $TMPFILE
-	rm -f $TMPFILE
 fi
```

Comment:
Should we add an `else` here so that we only run `DetectMain` once?

So that we run it either as detection or as confirmation of the given entry point.

Reply:
We probably want to align the Kotlin script then.

Reply:
Changed in latest version.